src/commands/notify.yml

```diff
@@ -24,6 +24,11 @@ parameters:
     type: enum
     enum: ["fail", "pass", "always"]
     default: "always"
+  branch_list:
```

Comment:
Might be more accurate to call this "branch_pattern"

Reply:
Fair point, I like it